edx_proctoring/backends/software_secure.py

```diff
@@ -360,7 +209,7 @@ def _create_zendesk_ticket(self, review, serialized_exam_object, serialized_atte
         """
```

Comment:
It looks like this pull request removes the one call to _create_zendesk_ticket, while also changing the function?

Reply:
I've moved the zendesk ticket creation to the review endpoint.